glue/__init__.py

```diff
@@ -118,6 +118,11 @@ def make_selector_func(roi):
     from .qt.custom_viewer import CustomViewer
     return CustomViewer.create_new_subclass(name, **kwargs)
 
+# Load built-in plugins
+from .config import load_plugins
+load_plugins()
```

Comment:
Another way to load the plugins would be to actually *not* have the plugins auto-register but have that instead be an app-specific thing. That is, maybe people building other glue apps don't want those plugins loaded by default. So as part of the glue *app* we could have an explicit loading of the plugins.